### DISCUSSION TITLE
Transparent PNG icons for the additional Rectangle Pro commands

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [rxhanson/RectanglePro-Community#618] [General]
# Request for Transparent PNG Icons for Rectangle Pro Commands

Could you provide transparent PNG icons for the additional Rectangle Pro commands, similar to the existing Rectangle icons? I’m working on a Raycast extension and would love to include them.  
[Rectangle Icons Repository](https://github.com/rxhanson/Rectangle/tree/main/Rectangle/Assets.xcassets)

---

## Missing Icons

Below is a list of specific actions that are missing icons (I don't have access to the src code of rectangle pro):

### 1. **Custom Positions**
- General Icon

### 2. **Active Layouts**
- General Icon

### 3. **Multiple Windows**
- Reveal Desktop Edge  
- 2x2 Tiles  
- 2x3 Tiles  
- Cascade All Windows  
- Cascade App Windows  

### 4. **Stash**
- Stash Left  
- Stash Right  
- Stash Up  
- Stash  
- Unstash  
- Unstash All

### 5. **Pin**
- Pin  
- Reflow Pin  

### 6. **Control**
- Close  
- Hide App  
- Minimize  
- Quit App

**Comment by rxhanson** [comment]
[Archive.zip](https://github.com/user-attachments/files/18137185/Archive.zip)

That should include everything. Apologies for the lack of organization in there.

**Reply by bigplayer-ai** [reply]
Hi @rxhanson,

Thank you for the icons archive! While integrating them into the Raycast extension, I noticed some inconsistencies and missing large icons. Here are the details:

Mismatched or Missing Icon Names:
	1.	Fill Category:
	•	large_fillCornersTLLargeTemplate.png → Should it be large_fillCornersTLTemplate.png?
	•	large_fillCornersTRLargeTemplate.png → Should it be large_fillCornersTRTemplate.png?
	•	Similar inconsistencies with BLLargeTemplate and BRLargeTemplate.
	2.	Fullscreen:
	•	Currently using window-positions/fullScreenTemplate.png. Is a large version available (e.g., large_fullScreenTemplate.png)?
	3.	Multiple Windows:
	•	Icons like revealDesktopEdge.png, tile2x2.png, tile2x3.png, cascadeApp.png, and cascadeAll.png—are large versions available (e.g., large_tile2x2.png)?
	4.	Stash Category:
	•	Unstash All uses unstashAllLargeTemplate.png, but Unstash uses large_restoreTemplate.png. Should these have consistent naming? also i noticed large_hideoutBottomTemplate instead of large_stashBottomTemplate.png for stash bottom/left/right/up...
	5.	Pin and Control Categories:
	•	Some icons (e.g., Pin, Reflow Pin, Close, Hide App) use non-large versions like pinTemplate.png. Are large versions available? Reflow pin uses toggleAllLargeTemplate.png

Thank you for your support and this great tool!
